config/peerpods/podvm/aws-podvm-image-handler.sh

```diff
@@ -207,9 +207,28 @@ function bootc_to_ami() {
     bootc_image_builder_conversion "${container_image_repo_url}" "${image_tag}" "${auth_json_file}" "${run_args}" "${bib_args}"
 }
 
+function prepare_for_prebuilt_artifact() {
+    echo "Preparing infrastructure for prebuilt artifact"
+
+    [[ ! ${BUCKET_NAME} ]] && export BUCKET_NAME=${AMI_BASE_NAME}-${AMI_VERSION}
+
+    # Update the BUCKET_NAME in the peer-pods-cm configmap
+    kubectl patch configmap peer-pods-cm -n openshift-sandboxed-containers-operator --type merge -p "{\"data\":{\"BUCKET_NAME\":\"${BUCKET_NAME}\"}}"
```

Comment:
Should the bucket name be in `peer-pods-cm` or `azure-podvm-image-cm` configMap ?

Reply:
good catch, yes, at this point it's better to be in the aws-podvm-image-cm